### PR TITLE
bug(1786733): added init.sql for firefox_accounts_derived.nonprod_fxa_auth_events_v1

### PR DESCRIPTION
**Pull request by kik-kik**
# bug(1786733): added init.sql for firefox_accounts_derived.nonprod_fxa_auth_events_v1

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

**Comment by kik-kik** [comment]
dry-run-sql failed with permission error to the table. Should be safe to ignore in this case.

**Comment by fbertsch** [comment]
> dry-run-sql failed with permission error to the table. Should be safe to ignore in this case.

Be sure to add it to the ignore list.

**Comment by kik-kik** [comment]
> > dry-run-sql failed with permission error to the table. Should be safe to ignore in this case.
> 
> Be sure to add it to the ignore list.

right, thanks for the reminder. I forgot about that. Should be part of the ignore/skip list.

File: sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_auth_events_v1/query.sql [hunk 1]
```diff
@@ -22,4 +24,4 @@ WHERE
   jsonPayload.type = 'amplitudeEvent'
   AND jsonPayload.fields.event_type IS NOT NULL
   AND jsonPayload.fields.user_id IS NOT NULL
-  AND _TABLE_SUFFIX = FORMAT_DATE('%y%m%d', @submission_date)
+  AND PARSE_DATE('%y%m%d', _TABLE_SUFFIX) = DATE(@submission_date)
```
Comment (fbertsch) [inline sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_auth_events_v1/query.sql:27]: Let's remove this change and keep it as it was
```suggestion
  AND _TABLE_SUFFIX = FORMAT_DATE('%y%m%d', @submission_date)
```

**Review by fbertsch — APPROVED** [review]
r+wc

**Pull request merged**